scripts/ruby_common/docker_utils.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
```

Comment:
this seems to be a copy from https://github.com/ChainSafe/forest/tree/main/scripts/sync_check/src, should we re-use it?

Reply:
Yes, I'd like the two services to share the ruby files. Not sure how to do it in this PR, though. Was gonna update `sync_check` in a different PR. Is there a sensible way to re-organize the code in this PR?

Reply:
I believe putting the Dockerfiles in parent, i.e. `scripts/` or making them use it as the context would allow playing with `require_relative` without many issues.

Reply:
That would mean changing (or refactoring) the files in `sync_check`, right? I wary about doing that in this PR as I'm not confident that I won't break anything.

Reply:
Yes. Otherwise you could depend on files in `sync_check`, but that wouldn't be much better. Feel free to change it inside `sync_check, then I can give you a hand at validating whether it doesn't break anything. Are you fine with that?

Reply:
I'm using `../ruby_common` as a bind in docker-compose. Not sure if that is allowed.

Reply:
I'll check it out.

Reply:
@lemmih In sync check compose, I added the common Ruby files to Ruby PATH, it should be fine now. I also moved the `sync_check.rb` back to `sync_check/`. Is it fine for you?

Reply:
Absolutely. It didn't belong in `ruby_common` but I didn't know how else to get the imports to work.